linux_os/guide/system/network/network-nftables/set_nftables_table/tests/nftables_incorrect_table_present.fail.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
```
jcerny@fedora:~/work/git/scap-security-guide (pr/11991)$ python3 tests/automatus.py rule --libvirt qemu:///system ssgts_rhel9 set_nftables_table
Setting console output to log level INFO
INFO - The data stream contains 3 Benchmarks
INFO - 0 - scap_org.open-scap_cref_ssg-rhel9-xccdf.xml
INFO - 1 - scap_org.open-scap_cref_rhel9-checks-sce-set_nftables_table.sh
INFO - 2 - scap_org.open-scap_cref_rhel9-checks-sce-ssh_keys_passphrase_protected.sh
INFO - Selected Benchmark is 0
INFO - To select a different Benchmark, use --xccdf-id-number option.
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/jcerny/work/git/scap-security-guide/logs/rule-custom-2024-05-13-1343/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_set_nftables_table
INFO - Script nftables_incorrect_family.fail.sh using profile (all) OK
ERROR - Script nftables_incorrect_table_present.fail.sh using profile (all) found issue:
ERROR - Rule evaluation resulted in pass, instead of expected fail during initial stage 
ERROR - The initial scan failed for rule 'xccdf_org.ssgproject.content_rule_set_nftables_table'.
INFO - Script nftables_no_tables.fail.sh using profile (all) OK
INFO - Script nftables_table_present.pass.sh using profile (all) OK
```

This scenario passes instead of the expected fail result.

I think the reason is that the check checks only the family, the check passes if `nft list tables | grep inet` returns a nonempty string.

Reply:
Let me take a look. This commit was long time only in my local repo... : )

Reply:
Unfortunately variables are not working in SCE checks and I don't have plans to work on this. I will remove this specific test scenario and include one more warning to the rule.